### PR TITLE
Cleanup actions config

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Actions runs are now faster than before, and have understandable job names. (This PR needs squashing)

image

image

### CODE REVIEW

**Review by owenvoke — APPROVED** [review]
Nice 👍 LGTM!

File: .github/workflows/static.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: 7.4
+        tools: composer:v2
+        coverage: none
+
+    - name: Install Dependencies
+      run: composer update --no-interaction --prefer-dist --no-progress
+
+    - name: Run Reactor
```
Comment (owenvoke) [inline .github/workflows/static.yml]:
```suggestion
    - name: Run Rector
```

Comment (GrahamCampbell) [reply .github/workflows/static.yml]: ☢️

**Comment by nunomaduro** [comment]
Thank you, again!

**Pull request merged**